src/core/list/components/Title.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const primary = {
+  fontSize: 36,
+  fontWeight: '700',
+  lineHeight: 1.11,
+  letterSpacing: '-0.03em',
```

Comment:
Usually, letter spacings are expressed in PX. (You wouldn't want your letter spacing to be relative to the screen size)

Reply:
Will handle this one in further issues.

Reply:
Can't we just do `letterSpacing: -1` now ?